mmengine/utils/misc.py

```diff
@@ -500,3 +500,47 @@ def wrapper(*args, **kwargs):
         return wrapper
 
     return decorator
+
+
+def locate(obj_name: str):
```

Comment:
```python
def locate(name: str)
    module_name, obj_name = string.rsplit('.', 1)
    module = importlib.import_module(module_name)
    obj = getattr(module, obj_name)
    return obj
```

Reply:
Both ways cannot locate the class attribute in one module since it is unknown that which part of the string is a module, and which is the imported object.  

BTW, the name `locate` is borrowed from the `pydoc.locate`